addons/sale_project/static/src/components/project_right_side_panel/components/project_profitability_section.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="sale_project.ProjectProfitabilitySection">
+        <tr>
+            <t t-set="revenue_label" t-value="props.labels[revenue.id] or revenue.id"/>
+            <td class="align-middle">
+                <t t-if="state.isFolded and isSectionFoldable">
+                    <button class="btn o_group_caret fa fa-fw fa-caret-right me-1" t-on-click="() => this.toggleSaleItems()"/>
+                </t>
+                <t t-if="!state.isFolded and isSectionFoldable">
+                    <button name="displaySaleItems" class="btn o_group_caret fa fa-fw fa-caret-down me-1" t-on-click="() => this.toggleSaleItems()"/>
+                </t>
```

Comment:
I would find it slightly more readable to factorize the `t-if="isSectionFoldable"` on a different indentation level.
Moreover, since the on click action is the same in both cases, we could have only one button, with conditional style